src/NUnitFramework/tests/nunit.framework.tests.csproj

```diff
@@ -11,11 +17,15 @@
     <!-- Microsoft.Bcl.Build workaround https://github.com/nunit/nunit/issues/2663 -->
     <SkipValidatePackageReferences>true</SkipValidatePackageReferences>
 
-    <UseWindowsForms>true</UseWindowsForms>
-    <UseWPF>true</UseWPF>
-
     <!-- Suppress Microsoft.NET.Sdk.WindowsDesktop warning that netcoreapp2.1 doesn't support desktop applications -->
-    <MSBuildWarningsAsMessages>$(MSBuildWarningsAsMessages);NETSDK1105</MSBuildWarningsAsMessages>
+    <!--<MSBuildWarningsAsMessages Condition="'$(TargetFramework)' == 'netcoreapp2.1'">$(MSBuildWarningsAsMessages);NETSDK1105</MSBuildWarningsAsMessages>-->
```

Comment:
We should probably remove this line and the line above, as we don't get warnings anymore (if I understand the code correctly).

Reply:
Ups. Left them by accident.